src/traces/scatter/attributes.js

```diff
@@ -67,7 +67,11 @@ module.exports = {
     },
     ids: {
         valType: 'data_array',
-        description: 'A list of keys for object constancy of data points during animation'
+        description: [
```

Comment:
@etpinard is there a way to automatically cast these to strings, or do I have to manually loop through the array, if provided, and cast them?